src/etaoin/api.clj

```diff
@@ -233,7 +233,7 @@
   [driver handle]
   (with-resp driver :post
     [:session (:session @driver) :window]
-    {:handle handle} _))
+    {:name handle} _))
```

Comment:
Thank you for pointing it. After scanning the source code of both Chrome and Gecko drivers, I found the following:

1. Chrome uses `name`, [source](https://github.com/bayandin/chromedriver/blob/master/client/chromedriver.py#L298)
2. Firefox uses `handler`, [source](https://searchfox.org/mozilla-central/source/testing/webdriver/src/command.rs#701)
3. The official W3C standard uses `handle` parameter, [source](https://www.w3.org/TR/webdriver/#switch-to-window)

So I believe, this function should be a multi-method. Let `:default` behavior uses `handler` parameter whereas `:chrome` and `:headless` use `:name`.